.github/workflows/publishing.yml

```
@@ -166,6 +166,7 @@ jobs:
 
             echo "Pushing $1 to $staging_branch"
             git add "$staging_folder" && git commit -m "$image_hash"
+            git pull origin/$staging_branch 2> /dev/null || true
```

I thought the remote repository had to be specified? Maybe it's inferred from the `origin/`?
```suggestion
            git pull origin origin/$staging_branch 2> /dev/null || true
```